### PR TITLE
Complete Migration from Bower to Yarn

### PR DESCRIPTION
**Pull request by bsimpson-r7**
The previous docker branch produced an error upon running due to the expiration of the Bower certificate. 
This merge completes the migration from Bower as package manager to Yarn, which enables Conqueso to run through Docker.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,9 @@
-FROM node:8
+FROM node:16
```
Comment (mbudde-r7) [inline Dockerfile:1]: Can we jump to node 18 here as its the most recent security patched version? https://endoflife.date/nodejs

Comment (ddavies-r7) [reply Dockerfile]: did you one better - node 20.

**Review by mbudde-r7 — COMMENTED** [review]
We should be commiting our `package-lock` here to maintain consitent deps

**Comment by ddavies-r7** [comment]
> We should be commiting our `package-lock` here to maintain consitent deps

How does this work if we want the image to be built on every developers machine to be used via docker. Do we still want the lock file added in there?

File: client/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "@bower_components/font-awesome": "FortAwesome/Font-Awesome#4.3.0",
+    "@bower_components/jquery": "jquery/jquery-dist#2.1.3",
+    "@bower_components/lodash": "lodash/lodash#3.3.1",
+    "@bower_components/moment": "moment/moment#2.9.0",
+    "@bower_components/require-handlebars-plugin": "SlexAxton/require-handlebars-plugin#0.11.2",
+    "@bower_components/requirejs": "jrburke/requirejs-bower#2.1.16",
+    "@bower_components/roboto-fontface": "choffmeister/roboto-fontface-bower#0.2.7",
+    "@bower_components/underscore": "jashkenas/underscore#>=1.5.0",
+    "cjs": "^0.0.11",
+    "express": "^4.18.2"
```
Comment (ndennison-r7) [inline client/package.json:14]: remove carats here
can add a .npmrc with [legacy deps](https://docs.npmjs.com/cli/v7/using-npm/config#legacy-peer-deps) to stop from happening in future

**Review by ddavies-r7 — APPROVED** [review]
Approving my own PR

                           🌔 
                          
                          
                          
            🚀

**Review by mbudde-r7 — APPROVED** [review]

**Pull request merged**